libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/metrics/FeedbackEvent.java

```diff
@@ -34,17 +34,17 @@
   public @interface FeedbackSource {
   }
 
-  public static final String FEEDBACK_TYPE_GENERAL_ISSUE = "general";
+  public static final String FEEDBACK_TYPE_OTHER_MAP_ISSUE = "other map issue";
```

Comment:
All of the changes to these strings are not necessary. The feedback types currently conform to the spec.